### ISSUE TITLE
Detail of the scores for MathVista

### ISSUE THREAD

**Issue by HugoLaurencon** [open-compass/VLMEvalKit#347]
Thank you for your useful library.

When I compute the scores for MathVista, I eventually obtain
` 0  Overall                      1000  687  584  68.7     58.4`
 Which one is the final score?
What does prefetch_hit mean?

Thank you for your help

**Comment by linhaojia13** [comment]
I meet the same problem.

**Comment by kennymckormick** [comment]
Hi, @HugoLaurencon @linhaojia13 ,
In our output file, `prefetch` means the prediction can be matched with the ground-truth answer (open-ended questions) or one of the choices (multi-choice questions, MCQ) with rule-based matching, it doesn't necessarily mean the prediction is correct for MCQ questions; while `hit` means the prediction is correct (the judgement is rely on GPT when `prefetch` fails). 
Basically, in the metric csv file, only `acc` matters and indicates the benchmark accuracy. We are sorry for the confusion and will try to improve the output format.

**Comment by andimarafioti** [comment]
Lol, I also had to google this xD Hi Hugo :)
